infra/main.bicep

```diff
@@ -77,6 +67,9 @@ param keyVaultResourceGroupName string = ''
 @description('Name of the Azure Log Analytics workspace')
 param logAnalyticsName string = ''
 
+@description('Location of the resource group for the OpenAI resources')
```

Comment:
@LittleLittleCloud we're defaulting to using the resource group location here? Will this still allow users to choose a different location for their resources?

Reply:
oops sorry